### PR TITLE
OpEx - remove websockets stack

### PR DESCRIPTION
**Pull request by codyseibert**
- replacing websockets stack with terraform + local solution

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-api/terraform/template/websockets/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  disconnectLambda,
+} = require('../../../src/notifications/disconnectLambda');
+
+exports.connectHandler = async event => {
+  return connectLambda(event);
+};
+
+exports.disconnectHandler = async event => {
+  return disconnectLambda(event);
+};
```
Comment (kkoskelin) [inline web-api/terraform/template/websockets/index.js]: nit: could these simply be expressed as `exports.connectHandler = connectLambda;` ?

File: web-api/websockets-local.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      response.writeHead(410);
+      response.end();
+    }
+  });
+});
+
+const PORT = 3011;
+
+server.listen(PORT, function () {
+  console.log(new Date() + ' Server is listening on port 8080');
```
Comment (kkoskelin) [inline web-api/websockets-local.js]: `listening on ${PORT}` ?

**Review by kkoskelin — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -166,27 +166,7 @@
...
-    "build:api:practitioners": "npm run build:_api -- web-api/src/practitionersHandlers.js",
-    "build:api:public": "npm run build:_api -- web-api/src/publicApiHandlers.js",
-    "build:api:reports": "npm run build:_api -- web-api/src/reportsHandlers.js",
-    "build:api:sections": "npm run build:_api -- web-api/src/sectionsHandlers.js",
-    "build:api:streams": "npm run build:_api -- web-api/src/streamsHandlers.js",
-    "build:api:trialSessions": "npm run build:_api -- web-api/src/trialSessionsHandlers.js",
-    "build:api:users": "npm run build:_api -- web-api/src/usersHandlers.js",
-    "build:api:workItems": "npm run build:_api -- web-api/src/workItemsHandlers.js",
-    "build:api": "npm run build:_api -- web-api/src/apiHandlers.js",
+    "build:all": "npm run clean && npm run build:assets && npm run build:client",
```
Comment (ericsorenson) [inline package.json:156]: Can the serverless libraries be removed?

Comment (codyseibert) [reply package.json:156]: yeah, I'll remove them

**Pull request merged**